src/pkg/packager/deploy.go

```diff
@@ -67,6 +71,11 @@ func (p *Packager) Deploy() (err error) {
 		return fmt.Errorf("deployment cancelled")
 	}
 
+	p.cfg.Pkg.Components, err = filter.Apply(p.cfg.Pkg)
```

Comment:
Feels a bit weird that we filter again in load then filter again here. I believe the only place that would matter, as the code is currently, is if we are deploying an OCI component without confirm, but since our filters have knowledge of confirm, perhaps we should call filters in OCISources either way and then skip the additional apply here

Reply:
fixed, was dead code from a previous change